types/noble-mac/tslint.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
```

Comment:
Sorry, for that. I was a bit confused, because there is just version 0.0.3 of `noble-mac` and so I had to specify version 0.0 as dt-header to fix that issue. Hope this is ok now.

Reply:
Yep, great